Introduction.adoc

```diff
@@ -1,26 +1,36 @@
+== Introduction
+
 ////
 I'd say this section is a good place to give a very sparse high level idea of
 what the synth is about, provide links to subsections, and provide a quickstart
 ////
 
-ZynAddSubFX is a powerful multi-platform open-source software sound synthesizer.
-Zyn provides a variety of sound creation options including 3 core synthesis
-engines, 10+ filter options, lfos/envelopes for modulation, microtonal support,
-multi-parameter automation support, as well as many other features.
-Zyn can be run both as a standalone application as well as a VST/LV2/DSSI plugin
-within your favorite DAW.
-ZynAddSubFX can be controlled through multiple interfaces, though this manual
-will focus on the new and powerful Zyn-Fusion interface.
+ZynAddSubFX is a multi-platform open-source software sound synthesizer, providing a variety of sound creation options including:
+
+* 3 core synthesis engines
+* 10+ filter options
+* LFOs/envelopes for modulation
+* Microtonal support
+* Multi-parameter automation support
```

Comment:
I'm torn on this. I'm a big fan of lists myself, but it does radically change the visual flow of the document. In this case I would likely stay away from changing things to a bulleted list as it does give it more of a specification feel